### DISCUSSION TITLE
Firefly iii integration in to home assistant

### DISCUSSION THREAD

**Discussion by ArchiRavi** [alexbelgium/hassio-addons#835] [General]
Hello, I have Installed Firefly iii as docker instance in my Proxmox, I would like to know how can I integrate that in to home assistant for some automation and alerting. Thanks

**Comment by alexbelgium** [comment]
Hi, for general questions on integrations you might have better chances of answer on the homeassistant community forum. Currently the only topic on firefly is this one : https://community.home-assistant.io/t/anyone-doing-anything-with-the-firefly-iii-api/433491 but it is still complex to implement.
